inc_internal/internal_model.h

```diff
@@ -152,7 +152,7 @@ ZITI_PR_BASE(XX, __VA_ARGS__)  \
 XX(path, model_string, none, path, __VA_ARGS__) \
 XX(is_running, model_bool, none, isRunning, __VA_ARGS__) \
 XX(hash, model_string, none, hash, __VA_ARGS__) \
-XX(signers, model_string, array, signerFingerprints, __VA_ARGS__)
+XX(signers, model_string, list, signerFingerprints, __VA_ARGS__)
```

Comment:
Could these container changes be backed out? It will create work in the swift SDK.

Reply:
this is internal model -- are you sure you're using/creating these in Swift?

Reply:
sorry, pls disregard. the `ziti_pr_process_cb` signature is all that the swift sdk cares about.